error: [Prusti: verification error] postcondition might not hold.
  --> $DIR/box.rs:8:11
   |
8  | #[ensures(false)]
   |           ^^^^^
   |
note: the error originates here
  --> $DIR/box.rs:9:1
   |
9  | / fn box_test(x: i32) {
10 | |     let y = Box::new(x);
11 | | }
   | |_^
note: counterexample for "x"
       value:   0
  --> $DIR/box.rs:9:13
   |
9  | fn box_test(x: i32) {
   |             ^
note: counterexample for "y"
       value:   box(-2147483610)
  --> $DIR/box.rs:10:13
   |
10 |     let y = Box::new(x);
   |             ^^^^^^^^^^^
note: counterexample for "result"
       value:   ()
  --> $DIR/box.rs:9:21
   |
9  |   fn box_test(x: i32) {
   |  _____________________^
10 | |     let y = Box::new(x);
11 | | }
   | |_^

error: aborting due to previous error

